contracts/EpochTokenLocker.sol

```diff
@@ -31,22 +31,22 @@ contract EpochTokenLocker {
         uint amount
     );
 
-    uint32 constant public BATCH_TIME = 300;
+    uint constant public BATCH_TIME = 5 minutes;
```

Comment:
This is a bonus of the PR, but I like it :)

Reply:
Ha ha, right yes. Although it seems we may revert this PR